shelf/hook/null_manager.py

```diff
@@ -0,0 +1,20 @@
+class NullManager(object):
```

Comment:
Since we have two managers using the same interface my brain is telling me that an interface would be helpful here but since we are cutting corners that probably doesn't make sense. Mentioning it anyways.